src/components/AppHeader/MobileMenu/BackButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { LegacyChevronLeft1pxIcon } from '@deriv/quill-icons';
+import { Text, useDevice } from '@deriv-com/ui';
+
+type TBackButton = {
+    buttonText: string;
+    onClick: () => void;
+};
+
+export const BackButton = ({ buttonText, onClick }: TBackButton) => {
+    const { isMobile } = useDevice();
+
+    return (
+        <button className='flex items-center w-full pt-[2rem] p-[3.2rem]' onClick={onClick}>
```

Comment:
if the padding isn't a decimal other than .5, most cases they already have the default there for it

```suggestion
        <button className='flex items-center w-full pt-8 p-[3.2rem]' onClick={onClick}>
```